framework/helpers/BaseHtml.php

```diff
@@ -1313,6 +1313,9 @@ public static function activeInput($type, $model, $attribute, $options = [])
         if (!array_key_exists('id', $options)) {
             $options['id'] = static::getInputId($model, $attribute);
         }
+        if (!array_key_exists('placeholder', $options) && $type!='hidden') {
```

Comment:
I think that the placeholder should be indicated.
Example ['placeholder' => true].

Reply:
Agree